docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,673 @@
+```
+
+Simulate data:
+```@example Main
+dat = simdat_crossed(subj_n,
+                     item_n,
+                     subj_btwn = subj_btwn,
+                     item_btwn = item_btwn,
+                     both_win = both_win);
+```
+
+Have a look:
+```@example Main
+first(DataFrame(dat),8)
+```
+The values we see in the column `dv` is just random noise.
+
+Set contrasts:
+```@example Main
+contrasts = Dict(:age => HelmertCoding(),
+                 :condition => HelmertCoding());
+```
+
+Define formula:
+```@example Main
+f1 = @formula dv ~ 1 + age * condition + (1|item) + (1|subj);
+```
+
+Fit the model:
+```@example Main
+m1 = fit(MixedModel, f1, dat, contrasts=contrasts)
+print(m1)
+```
+
+Because the `dv` is just random noise from N(0,1), there will be basically no subject or item random variance, residual variance will be near 1.0, and the estimates for all effects should be small.
+Don't worry, we'll specify fixed and random effects directly in `parametricbootstrap()`.
+
+
+Set random seed for reproducibility:
+```@example Main
+rng = StableRNG(42);
+```
+
+Specify `β`, `σ`, and `θ`, we just made up this parameter:
+```@example Main
+new_beta = [0., 0.25, 0.25, 0.]
+new_sigma = 2.0
+new_theta = [1.0, 1.0]
+```
+
+Run nsims iterations:
+```@example Main
+sim1 = parametricbootstrap(rng, nsims, m1,
+                        β = new_beta,
+                        σ = new_sigma,
+                        θ = new_theta,
+                        use_threads = false);
+```
+
+### Power calculation
+
+```@example Main
+ptbl= power_table(sim1)
+```
+
+For nicely displaying it, you can use pretty_table:
+```@example Main
+pretty_table(ptbl)
+```
+
+# 4. Recreate a more complex dataset from scratch and analyze power for specific model parameter but various sample sizes.
+
+# *4.1 Recreate the `kb07`-dataset from scratch*
+For full control over all parameters in our `kb07` data set we will recreate the design using the method shown above.
+
+
+Define subject and item number:
+```@example Main
+subj_n = 56
+item_n = 32
+```
+
+Define factors in a dict:
+```@example Main
+subj_btwn = nothing
+item_btwn = nothing
+both_win = Dict("spkr" => ["old", "new"],
+                "prec" => ["maintain", "break"],
+                "load" => ["yes", "no"]);
+```
+
+Simulate data:
+```@example Main
+fake_kb07 = simdat_crossed(subj_n, item_n,
+                     subj_btwn = subj_btwn,
+                     item_btwn = item_btwn,
+                     both_win = both_win);
+```
+
+Make a dataframe:
+```@example Main
+fake_kb07_df = DataFrame(fake_kb07)
+```
+
+Have a look:
+```@example Main
+first(fake_kb07_df,8)
+```
+
+The function `simdat_crossed` generates a fully crossed design.
+Unfortunately, our original design is not fully crossed. Every subject saw an image only once, thus in one of eight possible conditions. To simulate that we only keep one of every eight lines.
+
+TODO: NEED HELP, is that correct? or is it possible to do it in simdat_crossed?
```

Comment:
You can specify `subj_btwn` to get that -- it's still a balanced design (as long as you have sufficient items so that every condition is presented the same number of times total), but not "fully crossed", instead only "partially crossed".

(This is somewhat like the idea behind Latin Square designs.)

Reply:
Sorry but I didn't get that for 100% sure. 
If I put all my 3 factors to the subj_btwn, then the result indeed is a table of my wished length = 1972. BUT then for instance subject 01 see every item in the same condition. 
I think that is a problem for my LMM? 
What I would like to have is, that every subject see every item only once, but that all levels of the 3 factors appears to every subject.
Could you help me out, and give a hint, how to specify that in the simdat_crossed(), or do I need to delete extra rows and we can leave the procedure proposed in our tutorial?